app/views/care_plans/edit_status.js.erb

```diff
@@ -0,0 +1,4 @@
+console.log("<%= dom_id(@care_plan)%>_status");
+
+$("<%= dom_id(@care_plan)%>_status").replaceWith("<%= j(render('care_plans/forms/status', careplan: @care_plan)) %>");
```

Comment:
> I can't figure out why the page isn't reloading with the returned HTML form (it shows up that it was returned in the console). Any hints?

@WillSchwieder  you're missing the `#` part of the id selector. Without it, it won't identify the correct HTML element to replace.
```suggestion
$("#<%= dom_id(@care_plan)%>_status").replaceWith("<%= j(render('care_plans/forms/status', careplan: @care_plan)) %>");
```